docs/GettingStartedGuide.md

```diff
@@ -2,29 +2,29 @@
+1. Using your **platform credentials** that are available on the platform portal by means of the HERE OLP SDK for TypeScript authentication library.
+2. Using the **access key ID** and **access key secret** from the platform credentials as described in the [Authentication and Authorization Developer's Guide](https://developer.here.com/olp/documentation/access_control/topics/introduction.html) to create your own authentication and authorization client for retrieving HERE tokens that you can then provide to the HERE OLP SDK for C++ libraries.
 
-To obtain your platform credentials, create a new application via the [Apps and Keys](https://platform.here.com/profile/apps-and-keys) page. Then click **Create a key** to download these credentials. For more information on obtaining platform credentials, see the [Manage Apps](https://developer.here.com/olp/documentation/access-control/user-guide/topics/manage-apps.html) section of the [Teams and Permissions User Guide](https://developer.here.com/olp/documentation/access-control/user-guide/index.html).
+To get your platform credentials, on the [Apps and Keys](https://platform.here.com/profile/apps-and-keys) page, create a new application. Then, to download the credentials, click **Create a key**. For more information on how to get platform credentials, see the [Manage Apps](https://developer.here.com/olp/documentation/access-control/user-guide/topics/manage-apps.html) section of the [Teams and Permissions User Guide](https://developer.here.com/olp/documentation/access-control/user-guide/index.html).
 
 ## OLP Prerequisite Knowledge
 
-The use of the HERE OLP SDK for C++ requires a basic understanding of the following concepts related to the OLP:
+To use the HERE Open Location Platform (OLP) SDK forC++, you need to understand the following concepts related to OLP:
 
-* [OLP catalog](https://developer.here.com/olp/documentation/data-user-guide/shared_content/topics/olp/concepts/catalogs.html)
-* [OLP layers](https://developer.here.com/olp/documentation/data-user-guide/shared_content/topics/olp/concepts/layers.html)
-* [OLP partitions](https://developer.here.com/olp/documentation/data-user-guide/shared_content/topics/olp/concepts/partitions.html)
-* [HRN (HERE Resource Name)](https://developer.here.com/olp/documentation/data-user-guide/shared_content/topics/olp/concepts/hrn.html)
+* [Catalogs](https://developer.here.com/olp/documentation/data-user-guide/portal/layers/catalogs.html)
+* [Layers](https://developer.here.com/olp/documentation/data-user-guide/portal/layers/layers.html)
+* [Partitions](https://developer.here.com/olp/documentation/data-user-guide/portal/layers/partitions.html)
+* [HERE Resource Names (HRNs)](https://developer.here.com/olp/documentation/data-user-guide/shared_content/topics/olp/concepts/hrn.html)
```

Comment:
Why plural?

Reply:
There are several reasons:

1. To correspond to the section titles that the list items refer to.
2. Singular countable nouns require articles (indefinite in this case). It will be a bit confusing and will shift the emphasis from the key concepts if we start each bulleted item here with an indefinite article.

Reply:
Seems I mistakenly selected multiple lines. I only meant why plural for HRN.
At least for the abbreviation, I would still keep `HRN` without `s`.

Reply:
The acronym `HRN` stands for a singular noun phrase  `HERE Resource Name`. So if we make it plural, the acronym should also have the plural ending. For more info, please see [Add s to make an acronym plural](https://docs.microsoft.com/en-us/style-guide/acronyms#use-a-or-an-depending-on-pronunciation) in the Microsoft Manual of Style

Reply:
Ok, got it. Thanks.